src/status_im/utils/transducers.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.utils.transducers
+  "Utility namespace containing various usefull transducers"
+  (:require [taoensso.timbre :as log]))
```

Comment:
Not used?

Reply:
Good point, will remove !

Reply:
we have 400ms delay here, i doubt that it's possible to switch networks so fast. I doubt that it's possible just to navigate to profile that fast :)